streams.go

```diff
@@ -82,3 +87,20 @@ func (c *Client) GetFollowedStream(params *FollowedStreamsParams) (*StreamsRespo
 
 	return streams, nil
 }
+
+// GetStreamKey : Returns the secret stream key of the broadcaster
+//
+// Required scope: channel:read:stream_key
+func (c *Client) GetStreamKey(params *StreamKeyParams) (*StreamsResponse, error) {
+	resp, err := c.get("/streams/key", &ManyStreams{}, params)
+	if err != nil {
+		return nil, err
+	}
+
+	streams := &StreamsResponse{}
+	resp.HydrateResponseCommon(&streams.ResponseCommon)
+	streams.Data.Streams = resp.Data.(*ManyStreams).Streams
+	streams.Data.Pagination = resp.Data.(*ManyStreams).Pagination
```

Comment:
There's also no pagination for this endpoint according to the [docs](https://dev.twitch.tv/docs/api/reference#get-stream-key).